main/src/observables/factory.hpp

```diff
@@ -39,6 +39,9 @@
 #include "time_energies.hpp"
 #include "gravitational_waves.hpp"
 #include "wind_bubble_fraction.hpp"
+#ifdef USE_NUCLEAR_NETWORKS
+#include "nuclear_observable.hpp"
+#endif
```

Comment:
There is no real nuclear test case so I don't know where to reference nuclear_observable in the factory, but it will be done in the future. I don't think IO is the first thing to do in the integration.